packages/web-app-files/src/views/FilesDrop.vue

```diff
@@ -151,7 +152,7 @@ export default defineComponent({
         .catch((error) => {
           // likely missing password, redirect to public link password prompt
           if (error.statusCode === 401) {
-            return this.$authService.handleAuthError(this.$router.currentRoute)
```

Comment:
Is the authService not available anymore as `this.$authService`? 🤔

Reply:
Working with `prototype` seems to be the right way though: https://v2.vuejs.org/v2/cookbook/adding-instance-properties.html. I'll add it.